dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/BaseNameableObject.java

```diff
@@ -63,6 +63,10 @@
      */
     protected transient String displayDescription;
 
+    protected String formName;
```

Comment:
I think this might be okay. One issue is that most nameable objects do not have this property and then we end up with a lot of objects with an non-mapped (non-persisted) property which is not in use, which is confusing for users of the model. Thoughts?